recipe/build.sh

```diff
@@ -1,12 +1 @@
-export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
-export CXXLAGS="${CFLAGS}"
-export CPPFLAGS="-I${PREFIX}/include"
-export LDFLAGS="-L${PREFIX}/lib"
-
-if [[ `uname` == "Linux" ]]; then
-    export LDSHARED_FLAGS="-shared -pthread"
-else
-    export LDSHARED_FLAGS="-bundle -undefined dynamic_lookup"
-fi
-export LDSHARED="$CC $LDSHARED_FLAGS"
-LDSHARED=$LDSHARED $PYTHON -m pip install . --no-deps -vv
+$PYTHON -m pip install . --no-deps -vv
```

Comment:
Might even be worth getting rid of `build.sh` if it's only one line (in a future PR).